example_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 )
 
-func ExampleSingleThread() {
+func ExampleStopwatch_singleThread() {
```

Comment:
@ericychoi It's right in the 'testing' package godocs:
https://golang.org/pkg/testing
So, it's the 3rd form (Example(Type)_suffix) in the list below which is in the docs.
```
func Example_suffix() { ... }
func ExampleF_suffix() { ... }
func ExampleT_suffix() { ... }
func ExampleT_M_suffix() { ... }
```